docs/get-started/whats-a-story.md

```diff
@@ -38,7 +38,8 @@ The above story definition can be further improved to take advantage of [Storybo
   paths={[
     'react/button-story-with-args.js.mdx',
     'react/button-story-with-args.ts.mdx',
-    'vue/button-story-with-args.js.mdx'
+    'vue/button-story-with-args.js.mdx',
+    'angular/button-story-with-args.ts.mdx'
```

Comment:
```suggestion
    'angular/button-story-with-args.ts.mdx',
```

Reply:
Fixed with the latest commit. Thank you for checking it :)